shared/libraries/opcuatms/opcuatms_server/src/objects/tms_server_property.cpp

```diff
@@ -59,16 +62,21 @@ std::string TmsServerProperty::getBrowseName()
     return this->object.getName();
 }
 
+std::string TmsServerProperty::getNodeName()
```

Comment:
Can you check if we can avoid adding this method. Maybe we could try to override the existing getBrowseName() method.